tests/integrationtest/lib.go

```diff
@@ -2534,3 +2536,26 @@ func SetupLicense(license string) {
 	aviRestClientPool := cache.SharedAVIClients(lib.GetTenant())
 	lib.AKOControlConfig().SetLicenseType(aviRestClientPool.AviClient[0])
 }
+
+type ObjectNameMap struct {
+	nameMap map[string]int
+}
+
+func (o *ObjectNameMap) InitMap() {
+	o.nameMap = make(map[string]int)
+}
+func (o *ObjectNameMap) GenerateName(s string) string {
```

Comment:
this function along with the map is added to avoid specifying unique names to resources in new UTs.